web/viewer.css

```diff
@@ -1196,12 +1312,14 @@ html[dir='rtl'] .outlineItem > .outlineItems {
   text-decoration: none;
   display: inline-block;
   min-width: 95%;
-  min-width: calc(100% - 4px); /* Subtract the right padding (left, in RTL mode)
+  min-width: calc(
+    100% - 4px
+  ); /* Subtract the right padding (left, in RTL mode)
```

Comment:
The comment here is no longer aligned properly; and why even reformat this in the first place?

---

Given the overall size of this patch, please avoiding making unrelated changes.